src/Apps/Order/Components/OfferHistoryItem.tsx

```diff
@@ -0,0 +1,133 @@
+import { Button, Flex, FlexProps, Sans, Serif } from "@artsy/palette"
+import { OfferHistoryItem_order } from "__generated__/OfferHistoryItem_order.graphql"
+import { StaticCollapse } from "Components/StaticCollapse"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import {
+  StepSummaryItem,
+  StepSummaryItemProps,
+} from "Styleguide/Components/StepSummaryItem"
+
+class OfferHistoryItem extends React.Component<
+  {
+    order: OfferHistoryItem_order
+  } & StepSummaryItemProps,
+  { showingFullHistory: boolean }
+> {
+  state = { showingFullHistory: false }
+  render() {
+    const {
+      order: { totalListPrice, lastOffer, offers: _offers, buyer },
+      ...others
+    } = this.props
+    const { showingFullHistory } = this.state
+    const offers = _offers.edges.filter(
```

Comment:
Ah-- neat trick casting this as `_offers`. Could we either add a comment explaining what this filter is doing or rename the variable to `pastOffers` or something so it's clear?